README.md

```diff
@@ -1 +1,190 @@
-# inpycon2021-tasks
\ No newline at end of file
+# PyCon India 2021
+
+
+## Event Dates:
+- Conference: 16th and 17th Sep, 2021
+- Workshop: 18th Sep, 2021
+- DevSprint: 18th and 19th Sep, 2021
```

Comment:
There will be no devsprints, right?

Reply:
There will be devsprints, But we need to decide the format.